julia/LibCEED.jl/src/Operator.jl

```diff
@@ -123,3 +123,46 @@ function apply_add!(
 )
     C.CeedOperatorApplyAdd(op[], vin[], vout[], request[])
 end
+
+"""
+    assemble(op::Operator; request=RequestImmediate())
```

Comment:
I don't think it's intuitive for `assemble(operator)` to return the assembled representation of the QFunction. QFunction assembly isn't really intended for users anyway, so it might be preferable to omit this.

Reply:
Ok, I'll get rid of it.